source/options.css

```diff
@@ -2,12 +2,28 @@ body {
 	min-width: 400px; /* Required to size the Browser Action popup */
 }
 textarea {
+	display: block;
 	width: 100%;
+	font-family: monospace !important;
+	font-size: 12px;
+	line-height: 1.5;
+	resize: vertical;
+}
+label {
+	display: flex;
+	align-items: center;
+}
+label span {
+	margin-left: 0.5em;
 }
 .small {
 	font-size: 0.9em;
 }
 
+p {
+	margin: 0 0 1em;
+}
```

Comment:
`margin-bottom` already seems to be `1em` and I think `margin-top` looks better at `1em` (default)

Reply:
As you know vertical margins do collapse, and `margin-top` of `0` makes `h2` elements and `p` to be closer, making more content fit in view.

Reply:
Then we only need `margin-top: 0`